test/rubygems/test_gem_version.rb

```diff
@@ -41,6 +41,11 @@ def test_class_create
     assert_equal v('1.1'), Gem::Version.create(ver)
   end
 
+  def test_class_correct
```

Comment:
This test passes even without change in `lib/rubygems/version.rb`.

To test `true` and `false` is returned I suggest code similar to:

```ruby
  def test_class_correct
    assert_equal true, Gem::Version.correct?("5.1")
    assert_equal false, Gem::Version.correct?("an incorrect version")
  end
```

This test will fail without change in `lib/rubygems/version.rb` and pass with that change.